scripts/js/commands/checkAltText.ts

```diff
@@ -0,0 +1,97 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { globby } from "globby";
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+
+import { extractAlt } from "../lib/links/extractAltText.js";
+import { readMarkdown } from "../lib/markdownReader.js";
+
+interface Arguments {
+    [x: string]: unknown;
+  }
+
+const readArgs = (): Arguments => {
+    return yargs(hideBin(process.argv))
+      .version(false)
+      .parseSync();
+  };
+
+async function determineFiles(args: Arguments): Promise<[string[], string[]]> {
+  const mdGlobs = ["docs/guides/*.mdx", "docs/migration-guides/*.mdx"];
+  const notebookGlobs = ["docs/guides/*.ipynb", "docs/migration-guides/*.ipynb"];
+  return [await globby(mdGlobs), await globby(notebookGlobs)];
+}
+
+
+function getNullAlt(
```

Comment:
You won't need this function if you take my recommendation in `extractAltText.ts`